### ISSUE TITLE
Version 0.2.1 is not in NPM?

### ISSUE THREAD

**Issue by themacboy** [Turbo87/leaflet-sidebar#70]
Thx for your help !

**Comment by themacboy** [comment]
In CDNs is not deployed too hehehe ;)

**Comment by zedd45** [comment]
Sorry I missed this notification.  Let me look into that now.

**Comment by zedd45** [comment]
Ok, I had a few issues getting this done, including I apparently missed the `npm version patch` before, so I went ahead and published version `v0.2.2`.

```
npm notice === Tarball Details ===
npm notice name:          leaflet-sidebar
npm notice version:       0.2.2
npm notice package size:  669.0 kB
npm notice unpacked size: 709.7 kB
npm notice shasum:        d88e9efe9f2a84c39b97c79d0deb4acb8f785e08
npm notice integrity:     sha512-So29eH7CwC0P+[...]3kfseobkcffEA==
npm notice total files:   24
npm notice
+ leaflet-sidebar@0.2.2
```

This should be ready soon.  Let me know if this didn't work.
